src/DssSpellCollateralOnboarding.sol

```diff
@@ -35,50 +31,39 @@ contract DssSpellCollateralOnboardingAction {
     // A table of rates can be found at
     //    https://ipfs.io/ipfs/QmTRiQ3GqjCiRhh1ojzKzgScmSsiwQPLyjhgYSxZASQekj
     //
-    uint256 constant THREE_POINT_FIVE_PCT = 1000000001090862085746321732;
+    uint256 constant NUMBER_PCT = 1000000004090862085746321732;
```

Comment:
i wouldn't leave an actual percentage here in the example maybe `12345`

Reply:
done